StPetersburg/IVEf55.xml

```diff
@@ -0,0 +1,265 @@
+                              </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                          <locus from="11r" to="16r"></locus>
+                          <title type="incomplete" ref="LIT2000Mazmur"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <note>With musical notation</note>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                          <locus from="16v" to="37r"></locus>
+                          <title type="incomplete" ref="LIT2000Mazmur"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <note>With musical notation</note>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                          <locus from="39r" to="52v"></locus>
+                          <title type="incomplete" ref="LIT2509Weddas"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <note>With musical notation</note>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">54</measure>
+                                    <measure unit="leaf" type="blank">1</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>115</height>
+                                        <width>73</width>
+                                    </dimensions>
+                                </extent>
+
+                                  <condition key="good">
+                                    <note>Stains of water on <locus target="#37v #38r #38v"></locus></note>
+                                  </condition>
+
+                            </supportDesc>
+
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="15 18">
+
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                                <handNote script="Ethiopic" xml:id="h1">
+                                  <seg type="ink">Black, red.</seg>
+                                </handNote>
+                        </handDesc>
+                        <decoDesc>
+                          <decoNote xml:id="d1" type="miniature">
+                            <locus target="#2r"></locus>
+                            <desc>Partially colored miniature of <persName ref="PRS8886Sisinios">St Susǝnyos</persName> killing
```

Comment:
consider using the English form (Sisinnius) for a non-Ethiopic saint. (am updating the record)

He did not kill her, just pierced with a spear